### ISSUE TITLE
Creating an installation script for Bassa

### ISSUE THREAD

**Issue by amittal27** [scorelab/Bassa#364]
The instructions provide an outline for how to install Bassa onto the local machine; however, it may be hard to follow if the user is not already experienced with terminal, python, or git. Thus, it would be incredibly beneficial if the instructions were more detailed so users who aren't as familiar with the instructions specified in README.md for the installation procedure.

## Expected Behavior

I'm suggesting that we create installation scripts to make the process for installing Bassa onto one's machine much quicker and easier. In this way, the user doesn't need much technical experience to use Bassa. 

## Current Behavior

My suggestion for creating an installation script will allow the user to encounter fewer obstacles and eliminate frustration when installing Bassa.

## Possible Solution

I think the process for installing Bassa should be seamless. This can be done by creating installation scripts designated for specific operating systems, such as Windows (7, 8, 10), Mac OS X, and Linux (such as Ubuntu, Fedora, and Arch). Instead of having to follow a series of instructions (which do not provide all of the necessary details to install the software), the user just needs to download and run the installation script.

I have already started working on an installation script for installing Bassa onto a machine running Ubuntu 16.04. I would be happy to submit a pull request of the script after I refine it.

## Context

As I was working to install Bassa as one of the tasks for Google Code-in 2017, I ran into may difficulties that made the process lengthy and somewhat confusing. In other words, installing Bassa took a lot longer than necessary.

## Your Environment

* Operating System and version (desktop or mobile): Ubuntu 16.04 (virtual machine)

**Comment by kmehant** [comment]
Please look into this https://github.com/scorelab/Bassa/pull/374#issuecomment-498620595 
I am closing this issue for now.
